docker-compose up never starts server, fails with error in Django container

Following the instructions in the README:

1. `make dev-init`
2. `docker-compose up`

Version info:
- docker-compose version 1.23.2, build 1110ad01
- Docker version 18.09.3, build 774a1f4

```
erik@foundation:~/Code/securedrop.org$ cat .env
UID=1000
```

The django_1 container fails with the following error:

```
django_1      | ValueError: Unable to configure handler 'rotate': [Errno 13] Permission denied: '/django/logs/django-other.log'
securedroporg_django_1 exited with code 1
```

<details>
<summary>Full log output below.</summary>

```

erik@foundation:~/Code/securedrop.org$ docker-compose up
Starting securedroporg_node_1       ... done
Starting securedroporg_postgresql_1 ... done
Starting securedroporg_django_1     ... done
Attaching to securedroporg_node_1, securedroporg_postgresql_1, securedroporg_django_1
postgresql_1  | LOG:  database system was shut down at 2019-03-27 19:12:02 UTC
postgresql_1  | LOG:  MultiXact member wraparound protections are now enabled
django_1      | Waiting for postgres to start...
postgresql_1  | LOG:  database system is ready to accept connections
django_1      | Waiting for node to start...
postgresql_1  | LOG:  autovacuum launcher started
postgresql_1  | LOG:  incomplete startup packet
node_1        | npm WARN ajv-keywords@3.2.0 requires a peer of ajv@^6.0.0 but none is installed. You must install peer dependencies yourself.
node_1        | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.7 (node_modules/fsevents):
node_1        | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
node_1        | 
node_1        | audited 10146 packages in 14.817s
node_1        | found 665 vulnerabilities (325 low, 337 moderate, 3 high)
node_1        |   run `npm audit fix` to fix them, or `npm audit` for details
node_1        | 
node_1        | > securedrop@0.0.0 start /django
node_1        | > webpack -w -d --display-error-details
node_1        | 
django_1      | removed '.node_complete'
node_1        | Using STATIC_URL /static/
node_1        | 
node_1        | Webpack is watching the files…
node_1        | 
django_1      | Traceback (most recent call last):
django_1      |   File "/usr/local/lib/python3.4/logging/config.py", line 557, in configure
django_1      |     handler = self.configure_handler(handlers[name])
django_1      |   File "/usr/local/lib/python3.4/logging/config.py", line 725, in configure_handler
django_1      |     result = factory(**kwargs)
django_1      |   File "/usr/local/lib/python3.4/logging/handlers.py", line 150, in __init__
django_1      |     BaseRotatingHandler.__init__(self, filename, mode, encoding, delay)
django_1      |   File "/usr/local/lib/python3.4/logging/handlers.py", line 57, in __init__
django_1      |     logging.FileHandler.__init__(self, filename, mode, encoding, delay)
django_1      |   File "/usr/local/lib/python3.4/logging/__init__.py", line 1006, in __init__
django_1      |     StreamHandler.__init__(self, self._open())
django_1      |   File "/usr/local/lib/python3.4/logging/__init__.py", line 1035, in _open
django_1      |     return open(self.baseFilename, self.mode, encoding=self.encoding)
django_1      | PermissionError: [Errno 13] Permission denied: '/django/logs/django-other.log'
django_1      | 
django_1      | During handling of the above exception, another exception occurred:
django_1      | 
django_1      | Traceback (most recent call last):
django_1      |   File "./manage.py", line 12, in <module>
django_1      |     execute_from_command_line(sys.argv)
django_1      |   File "/usr/local/lib/python3.4/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
django_1      |     utility.execute()
django_1      |   File "/usr/local/lib/python3.4/site-packages/django/core/management/__init__.py", line 338, in execute
django_1      |     django.setup()
django_1      |   File "/usr/local/lib/python3.4/site-packages/django/__init__.py", line 22, in setup
django_1      |     configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
django_1      |   File "/usr/local/lib/python3.4/site-packages/django/utils/log.py", line 75, in configure_logging
django_1      |     logging_config_func(logging_settings)
django_1      |   File "/usr/local/lib/python3.4/logging/config.py", line 789, in dictConfig
django_1      |     dictConfigClass(config).configure()
django_1      |   File "/usr/local/lib/python3.4/logging/config.py", line 565, in configure
django_1      |     '%r: %s' % (name, e))
django_1      | ValueError: Unable to configure handler 'rotate': [Errno 13] Permission denied: '/django/logs/django-other.log'
securedroporg_django_1 exited with code 1
node_1        | Hash: c0d9bda78d8226f0757d
node_1        | Version: webpack 3.12.0
node_1        | Time: 4321ms
node_1        |      Asset     Size  Chunks                    Chunk Names
node_1        |  editor.js  2.29 MB       0  [emitted]  [big]  editor
node_1        |  common.js  2.24 MB       1  [emitted]  [big]  common
node_1        |     tor.js    31 kB       2  [emitted]         tor
node_1        | common.css   224 kB       1  [emitted]         common
node_1        | editor.css  3.74 kB       0  [emitted]         editor
node_1        |  [125] ./client/common/js/common.js 667 bytes {1} [built]
node_1        |  [144] ./client/common/sass/common.sass 41 bytes {1} [built]
node_1        |  [145] ./client/common/js/slidingnav.js 2.78 kB {1} [built]
node_1        |  [146] ./client/common/js/alert.js 677 bytes {1} [built]
node_1        |  [147] ./client/common/js/passwords.js 1.63 kB {1} [built]
node_1        |  [153] ./client/common/js/scanner.js 441 bytes {1} [built]
node_1        |  [154] ./client/autocomplete/js/editor.js 297 bytes {0} [built]
node_1        |  [155] ./client/autocomplete/sass/base.sass 41 bytes {0} [built]
node_1        |  [156] ./client/autocomplete/js/autocomplete.js 928 bytes {0} [built]
node_1        |  [263] ./client/tor/js/torEntry.js 264 bytes {2} [built]
node_1        |  [264] ./client/tor/js/detect-tor.js 3.68 kB {2} [built]
node_1        |  [265] ./node_modules/css-loader!./node_modules/postcss-loader/lib!./node_modules/sass-loader/lib/loader.js??ref--1-4!./client/common/sass/common.sass 231 kB [built]
node_1        |  [269] ./node_modules/css-loader!./node_modules/postcss-loader/lib!./node_modules/sass-loader/lib/loader.js??ref--1-4!./client/autocomplete/sass/base.sass 4.06 kB [built]
node_1        |     + 257 hidden modules
node_1        | Child extract-text-webpack-plugin node_modules/extract-text-webpack-plugin/dist node_modules/css-loader/index.js!node_modules/postcss-loader/lib/index.js!node_modules/sass-loader/lib/loader.js??ref--1-4!client/autocomplete/sass/base.sass:
node_1        |        [0] ./node_modules/css-loader!./node_modules/postcss-loader/lib!./node_modules/sass-loader/lib/loader.js??ref--1-4!./client/autocomplete/sass/base.sass 4.06 kB {0} [built]
node_1        |         + 1 hidden module
node_1        | Child extract-text-webpack-plugin node_modules/extract-text-webpack-plugin/dist node_modules/css-loader/index.js!node_modules/postcss-loader/lib/index.js!node_modules/sass-loader/lib/loader.js??ref--1-4!client/common/sass/common.sass:
node_1        |        [0] ./node_modules/css-loader!./node_modules/postcss-loader/lib!./node_modules/sass-loader/lib/loader.js??ref--1-4!./client/common/sass/common.sass 231 kB {0} [built]
node_1        |         + 1 hidden module
```
</details>

This has solved the initial django container failure (yay!), now I'm getting a new permission error from node which also causes the django container to never come up

```
django_1      | Waiting for postgres to start...
django_1      | Waiting for node to start...
node_1        | npm WARN checkPermissions Missing write access to /django/node_modules
node_1        | npm WARN ajv-keywords@3.2.0 requires a peer of ajv@^6.0.0 but none is installed. You must install peer dependencies yourself.
node_1        | 
node_1        | npm ERR! path /django/node_modules
node_1        | npm ERR! code EACCES
node_1        | npm ERR! errno -13
node_1        | npm ERR! syscall access
node_1        | npm ERR! Error: EACCES: permission denied, access '/django/node_modules'
node_1        | npm ERR!  { [Error: EACCES: permission denied, access '/django/node_modules']
node_1        | npm ERR!   stack:
node_1        | npm ERR!    'Error: EACCES: permission denied, access \'/django/node_modules\'',
node_1        | npm ERR!   errno: -13,
node_1        | npm ERR!   code: 'EACCES',
node_1        | npm ERR!   syscall: 'access',
node_1        | npm ERR!   path: '/django/node_modules' }
node_1        | npm ERR! 
node_1        | npm ERR! The operation was rejected by your operating system.
node_1        | npm ERR! It is likely you do not have the permissions to access this file as the current user
node_1        | npm ERR! 
node_1        | npm ERR! If you believe this might be a permissions issue, please double-check the
node_1        | npm ERR! permissions of the file and its containing directories, or try running
node_1        | npm ERR! the command again as root/Administrator (though this is not recommended).
node_1        | 
node_1        | npm ERR! A complete log of this run can be found in:
node_1        | npm ERR!     /home/docker_user/.npm/_logs/2019-03-27T21_00_31_735Z-debug.log
securedroporg_node_1 exited with code 243
```

--- a/docker-compose.yaml
+++ b/docker-compose.yaml
@@ -50,13 +50,12 @@ services:
       DJANGO_DB_NAME: securedropdb
       DJANGO_DB_PORT: 5432
       DJANGO_DB_HOST: db
-      DJANGO_LOG_PATH: /django-logs
+      DJANGO_LOG_PATH: /django/logs
       DJANGO_XMLTEST_OUTPUT: "yes"
       DEPLOY_ENV: dev
     working_dir: /django
     volumes:
       - ./:/django
-      - ./logs:/django-logs
     ports:
       - "127.0.0.1:8000:8000"
     networks:

